app/src/main/java/com/x8bit/bitwarden/data/vault/manager/VaultLockManagerImpl.kt

```diff
@@ -491,27 +486,37 @@ class VaultLockManagerImpl(
 
             VaultTimeout.OnAppRestart -> {
                 // If this is an app restart, trigger the timeout action; otherwise ignore.
-                if (checkTimeoutReason == CheckTimeoutReason.APP_RESTARTED) {
+                if (checkTimeoutReason is CheckTimeoutReason.AppCreated) {
                     // On restart the vault should be locked already but we may need to soft-logout
                     // the user.
```

Comment:
This doc likely needs updating because right now its only discussing the `firstTimeCreation` case.